### DISCUSSION TITLE
GUI button to delete all refresh tokens

### DISCUSSION THREAD

**Discussion by glitter-ball** [home-assistant/frontend#9085] [Other feature requests] [answered]
I'm an untidy person and don't log out of HA. So, I have hundreds (and hundreds) of refresh tokens showing in my profile page and deleting them one at a time is a major pain.

Feature request - in the Refresh Tokens section of the Profile page, provide a button to delete all refresh tokens other than that from the current session (which has a greyed out delete button anyway).

Alternatively/additionally... provide an option to delete tokens after a specified period.

Thanks!

**Comment by LeoLTM** [comment]
Oh yeah I would love to see that! 😅

**Comment by karwosts** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/18680
